src/ConfigManager.h

```diff
@@ -190,14 +206,32 @@ class ConfigManager {
                     ParameterMode mode) {
     parameters.push_back(new ConfigStringParameter(name, variable, size, mode));
   }
-  void save();
+
+  template <typename T>
+  void updateParameter(const char* name, T value) {
+    std::list<BaseParameter*>::iterator it;
+    for (it = parameters.begin(); it != parameters.end(); ++it) {
+      if ((*it)->getName() == name) {
+        DynamicJsonDocument doc(1024);
+        JsonObject json = doc.createNestedObject();
+        json[name] = value;
+        (*it)->fromJson(&json);
+        break;
```

Comment:
slightly hacky (?) ... couldnt get the Type definition into BaseParameter for the different types, so went the good old way of changing it to JSON and using the already set up update methods!!

Reply:
Disregard everything I wrote ... It appears I chased my tail, as there is some weird issue if you update a setting continually in the `loop` I thought it was the way I was doing updates/saves ... Ill not [squash this commit ](https://github.com/nrwiersma/ConfigManager/pull/86/commits/610c7ae310685b3b3ff15aabe89e6bcef2facefd#diff-3b7eeca402f95ff1c8eb045156a04d01R210) in the event you like this `updateParameter` hook ... it does abstract away from of how to handle the different types .. maybe ?

Reply:
The one benefit to this method is that it does the type casting for you .. and you cant try to jam a `random` as an `int` ... no bueno 

```
template <typename T>
  void updateParameter(const char* name, T value) {
    std::list<BaseParameter*>::iterator it;
    for (it = parameters.begin(); it != parameters.end(); ++it) {
      if ((*it)->getName() == name) {
        DynamicJsonDocument doc(1024);
        JsonObject json = doc.createNestedObject();
        json[name] = value;
        (*it)->fromJson(&json);
        break;
```